graphql/src/test/kotlin/com/trib3/graphql/execution/LeakyCauldronHooksTest.kt

```diff
@@ -202,36 +190,22 @@ class LeakyCauldronHooksTest {
                 .query("""query(${'$'}input: LocalDateTime!) {localDateTime(l:${'$'}input)}""")
                 .variables(mapOf("input" to "2019-10-30T00:01")).build()
         ).getData<Map<String, String>>()
-        assertThat(result["localDateTime"]).isEqualTo("2019-10-31T01:01")
+        assertThat(result["localDateTime"]).isEqualTo("2019-10-31T01:01:00")
     }
 
     @Test
     fun testLocalDate() {
         val result = graphQL.execute("""query {localDate(l:"2019-10-30")}""")
             .getData<Map<String, String>>()
         assertThat(result["localDate"]).isEqualTo("2019-10-31")
-        assertThat {
-            graphQL.execute("""query {localDate(l:123)}""")
-        }.isFailure().isInstanceOf(CoercingSerializeException::class)
-
-        assertThat {
-            graphQL.execute("""query {localDate(l:"123")}""")
-        }.isFailure().isInstanceOf(CoercingSerializeException::class)
-
-        assertThat {
-            LOCAL_DATE_SCALAR.coercing.serialize(123)
-        }.isFailure().isInstanceOf(CoercingSerializeException::class)
-
-        assertThat {
-            LOCAL_DATE_SCALAR.coercing.serialize(LocalDate.of(2019, 10, 31))
-        }.isSuccess().isEqualTo("2019-10-31")
```

Comment:
I removed direct tests of upstream scalar implementations (ie the ones calling `.serialize()` directly), but left in the ones that run by calling `graphQL.execute()` to ensure we're still hooking them up properly.  Arguably they're being overtested here, but also we'll catch any changes to their behavior (eg the type name changes got caught by the variable tests), so maybe this is actually what we want?

Reply:
I'm here to question if overtested is even a word 😆 I think I would agree that these weren't needed if we had more infrastructure in place to generate smoke tests to give us a lot of the same (and more) data. Since we don't I think it's reasonable to leave these in to help hopefully tip us off when changes are taking place.

Reply:
:joy: